src/main/java/net/dries007/tfc/objects/te/TEWorldItem.java

```diff
@@ -82,11 +82,12 @@ public SPacketUpdateTileEntity getUpdatePacket()
     @Override
     public NBTTagCompound getUpdateTag()
     {
-        // The tag from this method is used for the initial chunk packet, and it needs to have the TE position!
```

Comment:
I did it like so because I was afraid I might have broken something by doing that even though it souldn't and did not look like so. WorldItems left me with a scar.